stdlib/public/core/UnsafePointer.swift

```diff
@@ -996,7 +1249,23 @@ public struct UnsafeMutablePointer<Pointee>: _Pointer {
     Builtin.destroyArray(Pointee.self, _rawValue, count._builtinWordValue)
     return UnsafeMutableRawPointer(self)
   }
+}
 
+extension UnsafeMutablePointer {
+  // TODO: Merge this back into the noncopyable variant once we have @_preInverseGenerics
+  @available(swift, obsoleted: 6.0) // Legacy ABI compatibility
+  @usableFromInline
```

Comment:
```suggestion
  @discardableResult
  @usableFromInline
```
Is there a way to make a suggestion that merely inserts?

Reply:
I deliberately removed this annotation from this obsolete function, thinking it wouldn't be callable from anywhere -- however, `@available(swift, obsoleted: 6)` obviously did not prevent internal calls from getting routed to this overload, as the stdlib is not in fact being compiled in Swift 6 mode. 🤡 

The currrent solution should fix this thinko: `@available(swift,  obsoleted: 2)`. The `@_spi` also hides it from the regular swiftinterface and it will hopefully make some noise if some code still finds a way to call this. (Well, as long as it's inlinable.)